### ISSUE TITLE
Linter error incorrect

### ISSUE THREAD

**Issue by tomitrescak** [TypeStrong/atom-typescript#553]
I see that no one reported following error, how TS errors are reported in Linter.

![screen shot 2015-09-03 at 16 04 53](https://cloud.githubusercontent.com/assets/2682705/9651557/b3cac0c8-5255-11e5-9751-781fae2aea25.png)

**Comment by Cu3PO42** [comment]
I encountered this at one point, too, however I have not yet found a reliable way to reproduce it.

**Comment by nycdotnet** [comment]
I get it from time to time as well.  I'll try to look into it.

**Comment by nycdotnet** [comment]
I've found that "not assignable" errors tend to cause this.  I have a feeling that it's related to the error being multiline.  Take a look at the dev tools:

![lintererror-multiline](https://cloud.githubusercontent.com/assets/3755379/9666843/815c5aca-5246-11e5-8fe5-09d5ba695f5a.png)

**Comment by nycdotnet** [comment]
I've identified the issue.  I'll submit a PR later.

**Comment by nycdotnet** [comment]
This is a minimal repro:

```
interface myInterface {
  propertyA: string;
  propertyB: string;
}

var myObject1: myInterface = {
  propertyA: "test"
};
```

I've fixed the issue and published atom-typescript 6.0.2.  Please let me know if that fixes the issue for you too - and if so, please close the issue.  Thanks for the report.

**Comment by nycdotnet** [comment]
I believe this is resolved.  If you see it happen again, please re-open this ticket. Thanks!
